packages/components/src/local/asset-icon/styles.module.scss

```diff
@@ -91,12 +91,8 @@ $platforms: agi, auxiliary, boghammer, carrier, coastal-radar-site, datum,
 }
 
 .orientation {
-  background-image: url("./counters/orientation-marker.svg");
-  background-size: 80px;
-  fill: #f00;
-  color: #0f0;
-  filter: hue-rotate(244deg);
-  background-position: 25px 25px;
+  mask: url("./counters/orientation-marker.svg");
+  background-color: rgb(190, 48, 48);
```

Comment:
Aah, we're using RGB, but they are still hard-coded in the CSS.  Could you make it so we can specify the color in `index.tsx` please?
I'm pretty sure it will be specified in `#f48` notation.

Reply:
okay @IanMayo